pkgs/applications/networking/maestral/default.nix

```diff
@@ -1,24 +1,38 @@
-{ stdenv, lib, python3Packages, fetchFromGitHub
-, withGui ? false, wrapQtAppsHook ? null }:
+{ stdenv
+, lib
+, python3Packages
+, fetchFromGitHub
+, withGui ? false
+, wrapQtAppsHook ? null
+}:
 
 python3Packages.buildPythonApplication rec {
   pname = "maestral${lib.optionalString withGui "-gui"}";
-  version = "0.4.2";
+  version = "0.4.4";
 
   src = fetchFromGitHub {
     owner = "SamSchott";
     repo = "maestral-dropbox";
     rev = "v${version}";
-    sha256 = "0xis0cqfp3wgajwk44dmi2gbfirmz0a0zi25qxdzpdn0z19hp88m";
+    sha256 = "0br2njzbzb2qa7rj0biysnhgxgkdla88wbn7r3rwl9na0x91q1k4";
   };
 
   disabled = python3Packages.pythonOlder "3.6";
 
-  propagatedBuildInputs = (with python3Packages; [
-    blinker click dropbox keyring keyrings-alt Pyro4 requests u-msgpack-python watchdog
+  propagatedBuildInputs = with python3Packages; [
+    blinker
+    click
+    dropbox
+    keyring
+    keyrings-alt
+    Pyro5
+    requests
+    u-msgpack-python
+    watchdog
   ] ++ lib.optionals stdenv.isLinux [
-    sdnotify systemd
-  ] ++ lib.optional withGui pyqt5);
+    sdnotify
+    systemd
```

Comment:
not sure if propagating theses is the right thing to do, personally I would wrap them

Reply:
Why?